stream_alert/rule_processor/handler.py

```diff
@@ -54,8 +61,8 @@ def __init__(self, context, enable_alert_processor=True):
 
         self.metrics = Metrics('RuleProcessor', self.env['lambda_region'])
         self.enable_alert_processor = enable_alert_processor
-        self._failed_record_count = 0
-        self._alerts = []
+        self._failed_record_count = Value('I', 0)
```

Comment:
This is an example of why I personally prefer to import only top-level modules when possible. In other words, I prefer to just `import multiprocessing` instead of `from multiprocessing import Value`. Here, it's not clear where `Value` comes from; it's a very generic name. `multiprocessing.Value` would be more clear IMHO.

However, I know this is more consistent with the rest of the StreamAlert code. Ultimately up to you; just something to think about